src/guide/loops.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Loops
+---
+
+Loops allow you to repeat an action over and over (and over).
+
+## While Loops
```

Comment:
One thing I think these examples lack is either a button to demonstrate the output in the playground which isn't. a supported feature yet but an alternative would be just showing the printed output below so people can understand when the break occurs or continue occurs I have found it makes it a lot more clear for people who don't understand.

Reply:
Out of scope.